coderd/notifications/render/markdown.go

```diff
@@ -0,0 +1,113 @@
+package render
+
+import (
+	"bytes"
+	"strings"
+
+	"github.com/charmbracelet/glamour"
+	"github.com/charmbracelet/glamour/ansi"
+	gomarkdown "github.com/gomarkdown/markdown"
+	"github.com/gomarkdown/markdown/html"
+	"github.com/gomarkdown/markdown/parser"
+	"golang.org/x/xerrors"
+)
+
+// TODO: this is (mostly) a copy of coderd/parameter/renderer.go; unify them?
```

Comment:
AFAIR we use it to render plaintext descriptions. As long as there aren't any hidden corner cases, you can unify them 👍

Reply:
The only thing I changed was adding `parser.HardLineBreak` to the HTML output.
Cool, I'll unify.

Reply:
Fixed in b442b4e459820e0c2e436d9ce2dde2175a3d2777